### PR TITLE
Attestation creation

### PR DESCRIPTION
**Pull request by LiamClark**
Get rid of hard coded zkp.
Basic ui for attestation creation.
MetaData fields for now discarded

### CODE REVIEW

File: app/src/main/java/nl/tudelft/cs4160/identitychain/main/MainFragment.kt [hunk 1]
```diff
@@ -32,18 +34,58 @@ class MainFragment : Fragment() {
...
+        override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
+            return with(parent.context) {
+                linearLayout {
+                    textView(entries[position])
+                }
+            }
+        }
+
+        override fun getItem(position: Int): Any = entries[position]
+
```
Comment (LiamClark) [inline app/src/main/java/nl/tudelft/cs4160/identitychain/main/MainFragment.kt]: weird new lines

File: app/src/main/java/nl/tudelft/cs4160/identitychain/main/MainFragment.kt [hunk 2]
```diff
@@ -32,18 +34,58 @@ class MainFragment : Fragment() {
...
 
     companion object {
         private val TAG = MainFragment::class.java.toString()
     }
+
+    class SpinnerAdapter : BaseAdapter() {
+        val entries = listOf("age")
+        override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
+            return with(parent.context) {
```
Comment (LiamClark) [inline app/src/main/java/nl/tudelft/cs4160/identitychain/main/MainFragment.kt:70]: make single line

File: app/src/main/java/nl/tudelft/cs4160/identitychain/main/MainFragment.kt [hunk 3]
```diff
@@ -32,18 +34,58 @@ class MainFragment : Fragment() {
...
+        attestationType.adapter = SpinnerAdapter()
+
         view.addClaimButton.setOnClickListener {
-            val claimCreation = viewModel.createClaim()
-            if (claimCreation == null) {
-                Toast.makeText(activity, "Please select a peer to connect with", Toast.LENGTH_SHORT).show()
+            val parseNumbers = parseNumbers()
+            if (parseNumbers != null) {
+                val (a, b, m) = parseNumbers
+                val claimCreation = viewModel.createClaim(a, b, m)
```
Comment (LiamClark) [inline app/src/main/java/nl/tudelft/cs4160/identitychain/main/MainFragment.kt:43]: Saving the meta data will be adressed in a seperate pr

File: app/src/main/java/nl/tudelft/cs4160/identitychain/database/AttestationRequestRepository.kt [hunk 4]
```diff
@@ -1,38 +1,70 @@
...
     }
 }
 
 class FakeRepository : AttestationRequestRepository {
     val attestationRequests = ArrayList<AttestationRequest>()
+    val privateData = ArrayList<PrivateProof>()
+
+    override fun savePrivateData(private: SetupPrivateResult, seq_no: Int) {
+        privateData.add(PrivateProof.fromPrivateResult(private, seq_no))
+
```
Comment (LiamClark) [inline app/src/main/java/nl/tudelft/cs4160/identitychain/database/AttestationRequestRepository.kt]: weird new line

File: app/src/main/java/nl/tudelft/cs4160/identitychain/grpc/Operators.kt [hunk 5]
```diff
@@ -25,4 +26,10 @@ fun <A> ListenableFuture<A>.guavaAsSingle(scheduler: Scheduler): Single<A> {
             })
         })
     }
+}
+
+fun <T> startNetworkOnComputation(f: () -> Single<T>): Single<T> {
```
Comment (Griezzly) [inline app/src/main/java/nl/tudelft/cs4160/identitychain/grpc/Operators.kt]: what does this do exactly?

**Pull request merged**